recipes-kernel/linux/linux-armv7multi_4.14.bb

```diff
@@ -0,0 +1,33 @@
+# custom oe-based kernel for upstream plus RCN kernel patch
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}-${BRANCH_VERSION}:"
+
+include linux-armv7multi.inc
+
+SRC_URI = "git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git;protocol=http;branch=${KBRANCH};name=machine \
+           https://rcn-ee.net/deb/xenial-armhf/${RELEASE_TAG}-${BB_VERSION}/patch-${LINUX_VERSION}-${BB_VERSION}.diff.xz;name=patch \
```

Comment:
What contain this xenial-armhf patch?

Reply:
It's the usual RCN mainline patch from Robert @Digikey; see https://www.eewiki.net/display/linuxonarm/UDOO and his kernel build scripts on github.  The armv7-multi config supports several different boards and gets a lot of useful patches; I use the builder to test/port patches and push some missing ones to him (like the udoo patches, which are already in his v4.14.,x branch). Although in that particular branch his Udoo USB patches didn't get rebased onto 4.14 yet (he might be a little behind).

Reply:
Also that big patch is the same as the (separate) patches in the github repo so you can just pluck the ones you want instead of apply the giant diff.